extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -660,14 +920,20 @@ private Access authorizationCheck(final HttpServletRequest req, Action action)
     return access;
   }
 
+  // used for unit tests
+  Appenderator getAppenderator()
```

Comment:
Wanted to do that but https://github.com/druid-io/druid/pull/4874#discussion_r142752111 :)
Anyways I can add it

Reply:
Hah. Well I  that obviously the best option is if you're able to test this without relaxing visibility, but if not, then I think using the annotation is better than using a comment because at least it can be static analyzed.

Reply:
added `@VisibleForTesting`